projects/1.18/assets/stoneworks/stoneworks/lang/zh_cn.json

```
@@ -0,0 +1,584 @@
+  "block.stoneworks.diorite_bricks": "闪长岩砖",
+  "block.stoneworks.diorite_paver_slab": "闪长岩地砖台阶",
+  "block.stoneworks.diorite_paver_stairs": "闪长岩地砖楼梯",
+  "block.stoneworks.diorite_paver_wall": "闪长岩地砖墙",
+  "block.stoneworks.diorite_pavers": "闪长岩地砖",
+  "block.stoneworks.diorite_pillar": "闪长岩柱",
+  "block.stoneworks.diorite_plate_slab": "切制闪长岩台阶",
+  "block.stoneworks.diorite_plate_stairs": "切制闪长岩楼梯",
+  "block.stoneworks.diorite_plate_wall": "切制闪长岩墙",
+  "block.stoneworks.diorite_plates": "切制闪长岩",
+  "block.stoneworks.diorite_shingle_slab": "瓦制闪长岩台阶",
+  "block.stoneworks.diorite_shingle_stairs": "瓦制闪长岩楼梯",
+  "block.stoneworks.diorite_shingle_wall": "瓦制闪长岩墙",
+  "block.stoneworks.diorite_shingles": "瓦制闪长岩",
+  "block.stoneworks.diorite_tile_slab": "竹制闪长岩台阶",
+  "block.stoneworks.diorite_tile_stairs": "竹制闪长岩楼梯",
+  "block.stoneworks.diorite_tile_wall": "竹制闪长岩墙",
+  "block.stoneworks.diorite_tiles": "竹制闪长岩",
+  "block.stoneworks.end_stone_paver_slab": "末地石地砖台阶",
+  "block.stoneworks.end_stone_paver_stairs": "末地石地砖楼梯",
+  "block.stoneworks.end_stone_paver_wall": "末地石地砖墙",
+  "block.stoneworks.end_stone_pavers": "末地石地砖",
+  "block.stoneworks.end_stone_pillar": "末地石柱",
+  "block.stoneworks.end_stone_plate_slab": "切制末地石台阶",
+  "block.stoneworks.end_stone_plate_stairs": "切制末地石楼梯",
+  "block.stoneworks.end_stone_plate_wall": "切制末地石墙",
+  "block.stoneworks.end_stone_plates": "切制末地石",
+  "block.stoneworks.end_stone_shingle_slab": "瓦制末地石台阶",
+  "block.stoneworks.end_stone_shingle_stairs": "瓦制末地石楼梯",
+  "block.stoneworks.end_stone_shingle_wall": "瓦制末地石墙",
+  "block.stoneworks.end_stone_shingles": "瓦制末地石",
+  "block.stoneworks.end_stone_tile_slab": "竹制末地石台阶",
+  "block.stoneworks.end_stone_tile_stairs": "竹制末地石楼梯",
+  "block.stoneworks.end_stone_tile_wall": "竹制末地石墙",
+  "block.stoneworks.end_stone_tiles": "竹制末地石",
+  "block.stoneworks.granite_brick_slab": "花岗岩砖台阶",
+  "block.stoneworks.granite_brick_stairs": "花岗岩砖楼梯",
+  "block.stoneworks.granite_brick_wall": "花岗岩砖墙",
+  "block.stoneworks.granite_bricks": "花岗岩砖",
+  "block.stoneworks.granite_paver_slab": "花岗岩地砖台阶",
+  "block.stoneworks.granite_paver_stairs": "花岗岩地砖楼梯",
+  "block.stoneworks.granite_paver_wall": "花岗岩地砖墙",
+  "block.stoneworks.granite_pavers": "花岗岩地砖",
+  "block.stoneworks.granite_pillar": "花岗岩柱",
+  "block.stoneworks.granite_plate_slab": "切制花岗岩台阶",
+  "block.stoneworks.granite_plate_stairs": "切制花岗岩楼梯",
+  "block.stoneworks.granite_plate_wall": "切制花岗岩墙",
+  "block.stoneworks.granite_plates": "切制花岗岩",
+  "block.stoneworks.granite_shingle_slab": "瓦制花岗岩台阶",
+  "block.stoneworks.granite_shingle_stairs": "瓦制花岗岩楼梯",
+  "block.stoneworks.granite_shingle_wall": "瓦制花岗岩墙",
+  "block.stoneworks.granite_shingles": "瓦制花岗岩",
+  "block.stoneworks.granite_tile_slab": "竹制花岗岩台阶",
+  "block.stoneworks.granite_tile_stairs": "竹制花岗岩楼梯",
+  "block.stoneworks.granite_tile_wall": "竹制花岗岩墙",
+  "block.stoneworks.granite_tiles": "竹制花岗岩",
+  "block.stoneworks.mossy_andesite_brick_slab": "苔藓安山岩砖台阶",
```

```suggestion
  "block.stoneworks.mossy_andesite_brick_slab": "覆苔安山岩砖台阶",
```
应该是对应方块加上苔藓的变种
可根据此全文修改